msi/release.sh

```diff
@@ -62,6 +147,8 @@ BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
             </Component>
 EOF
 echo "$LIST" |
+sort |
+uniq |
```

Comment:
Well, if you want that change, maybe you could refactor it out into its own commit? It has little to do with added `release.sh` options...